agent/internal/taskoutput/test_log.go

```diff
@@ -93,6 +93,8 @@ func (h *testLogDirectoryHandler) run(ctx context.Context) error {
 	}
 	var fileChunks []fileChunk
 	ignore := filepath.Join(h.dir, testLogSpecFilename)
+	fileSizes := []int{}
```

Comment:
I think we should use int64 here instead of int (similar to @julianedwards comment). The `fileInfo.Size()` call below returns an int64 too

I read a little on the `int` type in golang rn, and it looks like it's system-dependent (sometimes 32, sometimes 64, etc). Or sometimes something between? It's quite weird